### ISSUE TITLE
Undefined offset: 0 in StackdriverExporter

### ISSUE THREAD

**Issue by TriPSs** [census-instrumentation/opencensus-php#106]
When the reporter does not need to post the spans i get the following error:
```
<b>Notice</b>:  Undefined offset: 0 in <b>/data/JIP-Accounts-API/vendor/opencensus/opencensus/src/Trace/Exporter/StackdriverExporter.php</b> on line <b>238</b><br />
```
But the error is gone when i for example update the probability sampler to 1.

My setup
```php
$reporter = new StackdriverExporter([
    "async"        => TRUE,
    "clientConfig" => $this->config,
]);

Tracer::start(
    $reporter,
    [
        "sampler" => new ProbabilitySampler(LOCAL ? 0 : 0.1), // sample 10% of requests
        "headers" => $this->headers,
    ]
);
```

Is this a bug in this lib or am i doing something wrong?

**Comment by chingor13** [comment]
Yep, this appears to be a bug. It looks like the exporter expects the request to have a root span and does not handle the case where it doesn't need to do anything.

**Comment by chingor13** [comment]
Thanks for the report! This should now be fixed and available in version 0.1.2
